syntax = "proto3";

package proto;

service Userer {
    rpc Login(LoginRequest) returns (LoginReply) {}
    rpc AddToFav(AddToFavRequest) returns (AddToFavReply) {}
}

message LoginRequest{
    string username = 1;
    string password = 2;
}


message LoginReply{
    bool correct = 1;
    string city1 = 2;
    string city2 = 3;
    string city3 = 4;
}

message AddToFavRequest{
    string username = 1;
    string city = 2;
}

message AddToFavReply{
    bool correct = 1;
}